### PR TITLE
Use typeface-roboto instead of google cdn

### PR DESCRIPTION
**Pull request by SebastianCApostolescu**
Less API calls.

Remember to `yarn`.

### CODE REVIEW

**Comment by pantharshit00** [comment]
Will it increase the bundle Size. Also roboto is usually cached in the device as it is very popular font from google cdn

**Comment by SebastianCApostolescu** [comment]
Indeed it will increase the bundle size, but after all it is still downloaded.
Anyway, the bundle is cached too and it allows the use of the font while offline. And also if I remember correctly the font face are already included.

**Comment by pantharshit00** [comment]
@SebastianCApostolescu you are correct MUI uses svg version on icons. @JesseRWeigel This change is worth committing.

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
   <title>Bulletin - Franciscan University of Steubenville</title>
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500|Material+Icons">
```
Comment (pantharshit00) [inline public/index.html]: Remember to test this as app might use material icons. @JesseRWeigel

Comment (pantharshit00) [inline public/index.html]: Remember to test this as app might use material icons. @JesseRWeigel

**Comment by JesseRWeigel** [comment]
Thanks @SebastianCApostolescu !

**Pull request merged**